src/Clients/CaradhrasCardClient.php

```diff
@@ -0,0 +1,484 @@
+     *
+     * @param  string  $cardId
+     * @param  string  $password
+     * @return object
+     * @throws \Illuminate\Http\Client\RequestException
+     */
+    public function setCardPassword(string $cardId): object
+    {
+        return $this->apiClient()
+            ->withHeaders(['senha' => str_pad(random_int(1000, 9999), 4, '0', STR_PAD_LEFT)])
+            ->post("/cartoes/{$cardId}/cadastrar-senha")
+            ->throw()
+            ->object();
+    }
+
+    /**
+     * Associate card to account.
+     *
+     * @param int $cardId
+     * @param int $accountId
+     * @param int $individualId
+     * @return bool
+     * @throws RequestException
+     */
+    public function associatePrepaidCardToAccount(int $cardId, int $accountId, int $individualId): bool
+    {
+        $this->apiClient()
+            ->post("/contas/{$accountId}/atribuir-cartao-prepago", [
+                'idCartao' => $cardId,
+                'idPessoaFisica' => $individualId,
+            ])
+            ->throw();
+
+        return true;
+    }
+
+    /**
+     * @param  int  $cardId
+     * @return Card
+     */
+    public function unlockSystemBlockedCard(int $cardId): Card
+    {
+        $unlockRequest = $this->apiClient()->post("/cartoes/{$cardId}/desbloquear-senha-incorreta");
+
+        return new Card($unlockRequest);
+    }
+
+    /**
+     * Update card password.
+     *
+     * @param  string  $cardId
+     * @param  string  $password
+     * @return array|object
+     */
+    public function updateCardPassword(string $cardId, $password)
+    {
+        return $this->apiClient()
+            ->withHeaders(['senha' => str_pad($password, 4, '0', STR_PAD_LEFT)])
+            ->put("/cartoes/{$cardId}/alterar-senha")
+            ->object();
+    }
+
+    /**
+     * Issue physical card.
+     *
+     * @param  int  $accountId
+     * @param  int  $individualId
+     * @return Card
+     * @throws \Illuminate\Http\Client\RequestException
+     * @throws \Exception
+     */
+    public function issuePhysicalCard(int $accountId, int $individualId): Card
+    {
+        $data = [
+            'id_pessoa' => $individualId,
+            'id_tipo_plastico' => config('caradhras.app.plastic_id'),
+        ];
+
+        $issueCardResponse = $this->apiClient(false)
+            ->post("/contas/{$accountId}/gerar-cartao-grafica", $data);
+
+        if ($issueCardResponse->failed()) {
+            throw new IssuePhysicalCardException();
+        }
+
+        $cardId = $issueCardResponse->json('idCartao');
+
+        $this->setCardPassword($cardId);
+
+        return $this->getCard($cardId);
+    }
+
+    /**
+     * Create virtual card.
+     *
+     * @param  int  $accountId
+     * @param  int  $individualId
+     * @return Card
+     */
+    public function issueVirtualCard(int $accountId, int $individualId): Card
+    {
+        $data = http_build_query([
+            'dataValidade' => now()->addYears(5)->format('Y-m-d\TH:i:s'),
```

Comment:
Eu sei que pedi pra mudar isso, mas a pouco descobri que o Carbon tem uma implementação feita para este padrão, prefiro utiliza-la

```suggestion
            'dataValidade' => now()->addYears(5)->toIso8601String
```